nck/project1
# changed readme

### to test this change

- pull down the change-readme branch
- run the code locally
- make sure the readme has the right text
